core/src/main/java/io/questdb/cairo/wal/seq/SeqTxnTracker.java

```diff
@@ -239,7 +156,7 @@ public synchronized boolean updateWriterTxns(long writerTxn, long dirtyWriterTxn
         // This is only called under TableWriter lock inside Apply2Wal job
         // with no threads race
         // TODO: remove other calls and make the call non-synchronized. The calls to reset txn
-        //       when queue is full seems like redundant after all the changes in CheckWalTransactionsJob
+        // when queue is full seems like redundant after all the changes in CheckWalTransactionsJob
```

Comment:
nit: this just messed up to-do highlghting, can we remove synchornized now?

Reply:
I removed the TODO and kept the lock in [ef5b832](https://github.com/questdb/questdb/pull/5347/commits/ef5b832b764a7c4e06e5042fc5139635f5e9bd34)